### DISCUSSION TITLE
Enhancements to go with beta launch

### DISCUSSION THREAD

**Discussion by rvcas** [aiken-lang/site#61] [Ideas]
Let's share some ideas to consider for the website.

**Comment by rvcas** [comment]
- show contributors some where
- roadmap

**Comment by KtorZ** [comment]
<img width="1123" alt="image" src="https://github.com/aiken-lang/site/assets/5680256/dc4c9eef-4081-4ddd-a813-be6f2a429f5e">

- Tracked tutorial steps, as seen on https://astro.build/

**Comment by KtorZ** [comment]
Make the "give feedback" link more visible, and turn it into "a button".

Currently:

<img width="346" alt="image" src="https://github.com/aiken-lang/site/assets/5680256/ba785a3f-8cd1-4116-9166-f341c1b4c43e">
 

What it could be:

<img width="235" alt="image" src="https://github.com/aiken-lang/site/assets/5680256/29e9e763-f7ae-4cda-9755-6a5bc52bfb9f">
